src/components/task-app/task-app.component.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import TaskForm from "../task-form";
+import TaskList from "../task-list";
+
+const TaskApp = () => {
+  const [toggleForm, setToggleForm] = useState(false);
+  const [taskList, setTaskList] = useState([]);
+
+  const openForm = () => {
+    setToggleForm(true);
+  };
+
+  const closeForm = () => {
+    setToggleForm(false);
+  };
+
+  const addNewTask = (data) => {
```

Comment:
let's try to represent data structures with a more specific name convention, in this case, `data` could be renamed to `task`

Reply:
0d5b5ec0578b86cc9d45a172265d8a3f1ba4d6f9 Data structure name changed to better represent its content.